packages/dev/core/src/Materials/effectRenderer.ts

```diff
@@ -343,15 +343,12 @@ export class EffectWrapper {
 
     /**
      * Disposes of the effect wrapper
-     * @param disposeEffect defines if the underlying effect should also be disposed (true by default)
      */
-    public dispose(disposeEffect = true) {
+    public dispose() {

```

Comment:
You changed the signature, which might be a breaking change? we can leave it and say it is unused. This is the reason for the build error

Reply:
sure!